commands/password-managers/bitwarden/copy-first-matching-totp.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Dependencies:
+#   1. The Bitwarden CLI: https://bitwarden.com/help/article/cli/
+#   2. The `jq` utility: https://stedolan.github.io/jq/
+#
+# Install via homebrew: `brew install bitwarden-cli jq`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Copy First Matching TOTP
+# @raycast.mode silent
+#
+# Optional parameters:
+# @raycast.packageName Bitwarden
+# @raycast.icon images/bitwarden.png
+# @raycast.argument1 { "type": "text", "placeholder": "Query" }
+#
+# Documentation
+# @raycast.author Adrian Reyes
+# @raycast.author Phil Salant
+# @raycast.authorURL https://github.com/int3rrupt
+# @raycast.description Search all items in a Bitwarden vault, and copy the TOTP of the first search result to the clipboard.
+
+itemNotFound() {
+  echo "The query '${BASH_ARGV[0]}' did not return any items."
+  exit 1
+}
+
+itemNoTotp() {
```

Comment:
To keep it consistent, I would rename it to "totpNotFound":

```suggestion
totpNotFound() {
```